app/views/layouts/application.html.haml

```diff
@@ -34,6 +35,11 @@
   %meta{ name: 'description', content: "#{@page_description.present? ? @page_description : 'GOV.UK Registers provides a secure API that is easy to integrate with so you can get the most up-to-date data from across government.'}" }
   = render partial: 'layouts/analytics'
 
+- content_for :after_header do
+  %aside{class: "notice"}
+    .container
+      %p{data: {"click-events" => "", "click-category" => "Content", "click-action" => "Notice link clicked"}} GOV.UK Registers will stop supporting data formats other than JSON or CSV from 30 September 2018. #{link_to "Read more about format changes", "/data-format-changes"}
```

Comment:
we can resolve the path using Rails `#{link_to "Read more about format changes", data_format_changes_path}`.

Reply:
Ah nice, 👌